lib/tasks/clean.rake

```diff
@@ -26,5 +26,17 @@ namespace :figgy do
 
       tus_storage.expire_files(expiration_date)
     end
+
+    desc "Clean old derivative processing files"
+    task derivative_artifacts: :environment do
+      all_files = Dir.glob("#{Dir.tmpdir}/derivative_generation")
```

Comment:
issue: I think this should be `Dir.glob("#{Dir.tmpdir}/derivative_generation/*")` Tried this in irb and the trailing `/*` was necessary for the match.

Reply:
Thanks - fixed.